ecms-social-integration/src/main/java/org/exoplatform/ecms/uploads/HTMLUploadImageProcessorImpl.java

```diff
@@ -224,6 +227,8 @@ public String processSpaceImages(String content, String spaceGroupId, String ima
           }
           if (imagesFolderNode.hasNode(folder)) {
             imagesFolderNode = imagesFolderNode.getNode(folder);
+            imagesFolderNode.addMixin("exo:privilegeable");
```

Comment:
Check if the node does not have that mixin before adding it

Reply:
done !